docs/development/creating-a-first-plugin.md

```diff
@@ -66,13 +63,13 @@ where the initialization and cleanup of the plugin will take place, as well as t
 registration of event handlers, new logic, etc. In lieu of a ```main()``` method, the plugin management system handles the initialization of plugins, so it is important that the annotations and types are setup correctly.
 
 ```java
-package me.plugincrafter.Demo;
+package io.gomint.testplugin;
```

Comment:
Please leave the package path as is. We'd like to embrace the reader to think of this as a possibility to use their own package path. I feel like `io.gomint.xxx` would be a mandatory package path they'd have to use.

Reply:
Although you can definitely make the `Demo` lowercase.